src/abstractdataframe/abstractdataframe.jl

```diff
@@ -405,31 +412,109 @@ describe(df)
 ```
 
 """
-StatsBase.describe(df::AbstractDataFrame) = describe(stdout, df)
-function StatsBase.describe(io, df::AbstractDataFrame)
-    for (name, col) in eachcol(df)
-        println(io, name)
-        describe(io, col)
-        println(io, )
+StatsBase.describe(df::AbstractDataFrame; kwargs...) = describe(stdout, df; kwargs...)
+function StatsBase.describe(io, df::AbstractDataFrame; colstats = [:mean, :min, :median, :max, :Nmissing, :datatype])
+    # Check that people don't specify the wrong fields. 
+    allowed_fields = [:mean, :sd, :min, :q25, :median, :q75, :max, :datatype, :Nunique, :Nmissing]
+    for i in colstats 
```

Comment:
You can do this more simply with `issubset`. Then it would be nice to print the name of problematic fields using `setdiff(colstats, allowed_fields)`. Also no need to use a new line for each name.